cypress/e2e/crypto/utils.ts

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright 2023 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import type { ISasEvent } from "matrix-js-sdk/src/crypto/verification/SAS";
+import type { MatrixClient } from "matrix-js-sdk/src/matrix";
+import type { VerificationRequest } from "matrix-js-sdk/src/crypto/verification/request/VerificationRequest";
+
+export type EmojiMapping = [emoji: string, name: string];
+
+/**
+ * wait for the given client to receive an incoming verification request
+ *
+ * @param cli - matrix client we expect to receive a request
+ */
+export function waitForVerificationRequest(cli: MatrixClient): Promise<VerificationRequest> {
+    return new Promise<VerificationRequest>((resolve) => {
+        const onVerificationRequestEvent = (request: VerificationRequest) => {
+            // @ts-ignore CryptoEvent is not exported to window.matrixcs; using the string value here
```

Comment:
Looks exported to me

![image](https://user-images.githubusercontent.com/2403652/233303270-89e792bc-e47d-4493-adef-e0036420994d.png)

Reply:
ugh but then I have to pass in `win` or `matrixcs` to `waitForVerificationRequest`.

This code isn't new, so I'm going to ignore this for now.